.codesandbox/sandbox/package.json

```diff
@@ -2,7 +2,6 @@
   "name": "solid-client-sandbox",
   "version": "1.0.0",
   "description": "Example project calling out to solid-client",
-  "main": "index.html",
```

Comment:
And for this it was complaining that it's not a JS file. And since it was more informative than actually useful, I just removed it :)